frontend/components/ManageStakingPage/StakingContractSection/MigrateButton.tsx

```diff
@@ -45,11 +46,10 @@ export const MigrateButton = ({
   const { service } = useService(serviceConfigId);
   const serviceTemplate = useMemo<ServiceTemplate | undefined>(
     () =>
-      service
-        ? getServiceTemplate(service.hash)
-        : SERVICE_TEMPLATES.find(
-            (template) => template.agentType === selectedAgentType,
-          ),
+      (service && getServiceTemplate(service.hash)) ??
```

Comment:
found a error here when I had a service with old hash, but template was updated with a new one, I was unable to migrate as `getServiceTemplate(service.hash)` returned undefined and we check `serviceTemplate` in migrate handler